nmigen_boards/de0_nano.py

```diff
@@ -0,0 +1,73 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.intel import *
+from .resources import *
+
+__all__ = ["DE0NanoPlatform"]
+
+
+class DE0NanoPlatform(IntelPlatform):
+    device      = "EP4CE22" # Cyclone IV 22K LEs
+    package     = "F17"   # FBGA-256
+    speed       = "C6"
+    default_clk = "clk50"
+    resources   = [
+        Resource("clk50", 0, Pins("R8", dir="i"),
+                 Clock(50e6), Attrs(io_standard="3.3-V LVTTL")),
+
+        Resource("i2c_flash", 0,
```

Comment:
Should this be using `I2CResources`?

Reply:
no idea, would that be easier? It's a normal I2C interface to a 256x8 eeprom

Reply:
fixed